src/native.js

```diff
@@ -197,6 +217,28 @@ export function getAssetMessage(data, adObject) {
   return message;
 }
 
+export function getAllAssetsMessage(data, adObject) {
+  const message = {
+    message: 'assetResponse',
+    adId: data.adId,
+    assets: []
+  };
+
+  Object.keys(adObject.native).forEach(function(key, index) {
+    if (key === 'adTemplate' && adObject.native[key]) {
+      message.adTemplate = getAssetValue(adObject.native[key]);
+    } else if (key === 'rendererUrl' && adObject.native[key]) {
+      message.rendererUrl = getAssetValue(adObject.native[key]);
```

Comment:
So these values are being sent through to the top level of the message, not within the assets array?

What is the reason for this?

Reply:
I placed them at the top level instead of the assets as the assets are looped through and replace their corresponding macro with a value. These two will never need use that functionality and if exist are simply utilized for the replacement themselves. I figured separating them made more sense, but if opposed, they can be added to the assets with other logic in place to detect in the PUC.

Reply:
Nope! Good idea!